src/main/java/view/EpisodeView.java

```diff
@@ -0,0 +1,116 @@
+package view;
+
+import entities.Episode;
+import entities.TextChunk;
+import interface_adapter.display_episode.DisplayEpisodeState;
+import interface_adapter.display_episode.DisplayEpisodeViewModel;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.beans.PropertyChangeEvent;
+import java.beans.PropertyChangeListener;
+
+public class EpisodeView extends JPanel implements ActionListener, PropertyChangeListener {
```

Comment:
Why does the View itself implement ActionListener? Shouldn't there just be a button that has an ActionListener for playing the audio?

Reply:
The view implements ActionListener merely for the actionPerfomred method